src/test/java/org/jabref/logic/bst/BibtexCaseChangersTest.java

```diff
@@ -1,134 +1,148 @@
+
+                Arguments.of("HALLO", "hallo"),
+                Arguments.of("HALLO", "HAllo"),
+                Arguments.of("HALLO WORLD", "HAllo World"),
+                Arguments.of("HALLO WORLD. HOW", "HAllo World. How"),
+                Arguments.of("HALLO {worLD}. HOW", "HAllo {worLD}. how"),
+                Arguments.of("HALLO {\\WORLD}. HOW", "HAllo {\\woRld}. hoW"),
+
+                // testSpecialBracketPlacement
+                Arguments.of("AN {$O(n \\log n)$} SORTING ALGORITHM", "An {$O(n \\log n)$} Sorting Algorithm")
+        );
     }
 
-    private void assertCaseChangerTitleLowers(final String string, final String string2) {
-        assertEquals(string, BibtexCaseChanger.changeCase(string2, FORMAT_MODE.TITLE_LOWERS));
+    @ParameterizedTest
+    @MethodSource("provideTitleCaseAllLowers")
+    public void testTitleCaseAllLowers(String expected, String toBeFormatted) {
+        assertEquals(expected, BibtexCaseChanger.changeCase(toBeFormatted, FORMAT_MODE.ALL_LOWERS));
     }
 
-    private void assertCaseChangerAllLowers(final String string, final String string2) {
-        assertEquals(string, BibtexCaseChanger.changeCase(string2, FORMAT_MODE.ALL_LOWERS));
+    private static Stream<Arguments> provideTitleCaseAllLowers() {
+        return Stream.of(
+                // CaseChangers.TITLE is good at keeping some words lower case
+                // Here some modified test cases to show that escaping with BibtexCaseChanger also works
+                // Examples taken from https://github.com/JabRef/jabref/pull/176#issuecomment-142723792
+                Arguments.of("this is a simple example {TITLE}", "This is a simple example {TITLE}"),
+                Arguments.of("this {IS} another simple example tit{LE}", "This {IS} another simple example tit{LE}"),
+                Arguments.of("{What ABOUT thIS} one?", "{What ABOUT thIS} one?"),
+                Arguments.of("{And {thIS} might {a{lso}} be possible}", "{And {thIS} might {a{lso}} be possible}")
+        );
     }
 
-    private void assertCaseChangerAllUppers(final String string, final String string2) {
-        assertEquals(string, BibtexCaseChanger.changeCase(string2, FORMAT_MODE.ALL_UPPERS));
+    @Disabled
+    @Test
+    public void testTitleCaseAllUppers() {
+        /* the real test would look like as follows. Also from the comment of issue 176, order reversed as the "should be" comes first */
+        // assertCaseChangerTitleUppers("This is a Simple Example {TITLE}", "This is a simple example {TITLE}");
+        // assertCaseChangerTitleUppers("This {IS} Another Simple Example Tit{LE}", "This {IS} another simple example tit{LE}");
+        // assertCaseChangerTitleUppers("{What ABOUT thIS} one?", "{What ABOUT thIS} one?");
+        // assertCaseChangerTitleUppers("{And {thIS} might {a{lso}} be possible}", "{And {thIS} might {a{lso}} be possible}")
```

Comment:
In case a test is `@Disabled`, the contents of the test can go enabled. Since this is probably the last comment on this PR, I'll let it go and we will fix for ourselves.